packages/common/src/dotlottie-player.ts

```diff
@@ -1694,10 +1527,20 @@ export class DotLottiePlayer {
         throw createError('no animation to load!');
       }
 
+      // TODO: Get states from dotlottie-js once its ready.
+      const loadDemoState = ['pigeon', 'wifi', 'smiley'].every((key) => Array.from(animations.keys()).includes(key));
```

Comment:
Even though the states are hard coded in the example state file, we dont need to hard code them in to common code too no ? We can get them from manifest without using 'pigeon' 'wifi' and 'smiley'?